src/workerd/io/worker-interface.capnp

```diff
@@ -15,6 +15,16 @@ using import "/capnp/compat/byte-stream.capnp".ByteStream;
 using import "/workerd/io/outcome.capnp".EventOutcome;
 using import "/workerd/io/script-version.capnp".ScriptVersion;
 
+struct InvocationSpanContext {
+  struct TraceId {
+    high @0 :UInt64;
```

Comment:
This reminds me: high and low are an eyesore, we should really implement the TODO to use `__int128_t` some day.

Reply:
Agree. For the time being tho, let's leave that as a todo.